### PR TITLE
maint: update authors file

### PR DESCRIPTION
**Pull request by oscarbenjamin**
#### Release Notes

NO ENTRY

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -999,12 +999,59 @@ Shubham Agrawal <shubham.ag6845@gmail.com>
...
+gopalamn <gopalamn@umich.edu>
+Kartik Sethi <kartiks31416@gmail.com>
+MaqOwais <quadirowais200@gmail.com>
+Harshit Yadav <harshityadav2k@gmail.com>
+Sidharth Mundhra <sidharthmundhra16@gmail.com>
+suryam35 <suryamkalra35@gmail.com>
+LAPTOP-AS1M2R8B\codem <codemastercpp@gmail.com>
+kunal singh <ksingh19136@gmail.com>
+mayank raj <mayank_1901cs35@iitp.ac.in>
+mayank <54908605+mayankray2020@users.noreply.github.com>
```
Comment (oscarbenjamin) [inline AUTHORS:1036]: I guess these are the same person

File: AUTHORS [hunk 2]
```diff
@@ -999,12 +999,59 @@ Shubham Agrawal <shubham.ag6845@gmail.com>
...
+JSS95 <52185322+JSS95@users.noreply.github.com>
```
Comment (oscarbenjamin) [inline AUTHORS:1037]: @jss95 should I change this? Are you already in the AUTHORS file under a different name?

File: AUTHORS [hunk 3]
```diff
@@ -999,12 +999,59 @@ Shubham Agrawal <shubham.ag6845@gmail.com>
...
+Saket Kumar Singh <saketkumar1202@gmail.com>
+muzimuzhi <muzimuzhi@gmail.com>
+eagleoflqj <liumeo@pku.edu.cn>
+Paul Mandel <paulmandel@google.com>
+Nisarg Chaudhari <54911392+Nisarg-Chaudhari@users.noreply.github.com>
+Dominik Stańczak <stanczakdominik@gmail.com>
+Rodrigo Luger <rodluger@gmail.com>
+me <none@none>
+Marco Antônio Habitzreuter <mahabitzreuter@gmail.com>
+Ayush <bisht.ayush2001@gmail.com>
```
Comment (ayushbisht2001) [inline AUTHORS:1055]: Ayush Bisht  <bisht.ayush2001@gmail.com>

File: AUTHORS [hunk 4]
```diff
@@ -999,12 +999,59 @@ Shubham Agrawal <shubham.ag6845@gmail.com>
...
+akshanshbhatt <53227127+akshanshbhatt@users.noreply.github.com>
+ayushbisht2001 <61404154+ayushbisht2001@users.noreply.github.com>
```
Comment (ayushbisht2001) [inline AUTHORS:1057]: same person

File: AUTHORS (hunk 2 above)
Comment (oscarbenjamin) [reply AUTHORS:1036]: I see `Jisoo Song` as well. Should I alias them all to that?

Comment (oscarbenjamin) [reply AUTHORS]: I've aliased them to that.

File: AUTHORS (hunk 4 above)
Comment (oscarbenjamin) [reply AUTHORS]: Thanks. I think I've fixed that. Does it look okay now?

File: AUTHORS (hunk 3 above)
Comment (oscarbenjamin) [reply AUTHORS]: Done

File: AUTHORS [hunk 5]
```diff
@@ -999,12 +998,55 @@ Shubham Agrawal <shubham.ag6845@gmail.com>
...
 Amanda Dsouza <meezamanda@yahoo.com>
 Nijso Beishuizen <nijso@koolmees.numerically-related.com>
+Harry Zheng <harry@harryzheng.com>
+Felix Yan <felixonmars@archlinux.org>
+Constantin Mateescu <costica1234@me.com>
+Eva <eva.tiwari@gmail.com>
+Aditya Kumar Sinha <adityakumar113141@gmail.com>
+Soumi Bardhan <51290447+Soumi7@users.noreply.github.com>
+Kaustubh <ckaustubhm06@gmail.com>
+Kristianmitk <hello@kristianbrunn.com>
```
Comment (kristianmitk) [inline AUTHORS:1017]:
```suggestion
Kristian Brünn <hello@kristianbrunn.com>
```

Comment (oscarbenjamin) [reply AUTHORS:1017]: Thanks

**Comment by oscarbenjamin** [comment]
I've emailed quite a few of the people listed here asking what name they want recorded in the AUTHORS file and I'm just waiting to hear back.

File: AUTHORS [hunk 6]
```diff
@@ -999,12 +999,59 @@ Shubham Agrawal <shubham.ag6845@gmail.com>
...
+muzimuzhi <muzimuzhi@gmail.com>
+eagleoflqj <liumeo@pku.edu.cn>
+Paul Mandel <paulmandel@google.com>
+Nisarg Chaudhari <54911392+Nisarg-Chaudhari@users.noreply.github.com>
+Dominik Stańczak <stanczakdominik@gmail.com>
+Rodrigo Luger <rodluger@gmail.com>
+me <none@none>
+Marco Antônio Habitzreuter <mahabitzreuter@gmail.com>
+Ayush <bisht.ayush2001@gmail.com>
+akshanshbhatt <53227127+akshanshbhatt@users.noreply.github.com>
```
Comment (akshanshbhatt) [inline AUTHORS:1052]:
```suggestion
Akshansh Bhatt <akshansh@tuta.io>
```

Comment (oscarbenjamin) [reply AUTHORS]: Thanks

File: AUTHORS [hunk 7]
```diff
@@ -999,12 +998,55 @@ Shubham Agrawal <shubham.ag6845@gmail.com>
...
+Akshat Sood <68052998+akshatsood2249@users.noreply.github.com>
+Jose M. Gomez <chemoki@gmail.com>
+Stefan Petrea <stefan@garage-coding.com>
+Praveen Sahu <povinsahu@gmail.com>
+Mark Bell <mark00bell@googlemail.com>
+AlexCQY <alex_chua@u.nus.edu>
+Fabian Froehlich <fabian@schaluck.com>
+Nikhil Gopalam <gopalamn@umich.edu>
+Kartik Sethi <kartiks31416@gmail.com>
+MaqOwais <quadirowais200@gmail.com>
```
Comment (oscarbenjamin) [inline AUTHORS:1028]: @MaqOwais how should your name be recorded here?

File: AUTHORS [hunk 8]
```diff
@@ -999,12 +998,55 @@ Shubham Agrawal <shubham.ag6845@gmail.com>
...
+Harshit Yadav <harshityadav2k@gmail.com>
+Sidharth Mundhra <sidharthmundhra16@gmail.com>
+Suryam Arnav Kalra <suryamkalra35@gmail.com>
+LAPTOP-AS1M2R8B\codem <codemastercpp@gmail.com>
```
Comment (oscarbenjamin) [inline AUTHORS:1032]: @prince776 How should your name be recorded here?

File: AUTHORS [hunk 9]
```diff
@@ -999,12 +998,55 @@ Shubham Agrawal <shubham.ag6845@gmail.com>
...
+Kunal Singh <ksingh19136@gmail.com>
+mayank raj <mayank_1901cs35@iitp.ac.in>
+Achal Jain <2achaljain@gmail.com>
+Mario Maio <mario.maio@aruba.it>
+Aaron Stiff <69512633+AaronStiff@users.noreply.github.com>
+Wyatt Peak <wyattpeak@gmail.com>
+BhaskarJoshi-01 <bhaskar.joshi@research.iiit.ac.in>
+Aditya Jindal <jaditya8889@gmail.com>
+VBhat97 <vaibhav.bhat2097@gmail.com>
```
Comment (oscarbenjamin) [inline AUTHORS:1041]: @VBhat97 how should your name be recorded here?

File: AUTHORS (hunk 8 above)
Comment (prince776) [reply AUTHORS:1032]: Prince Gupta

Comment (oscarbenjamin) [reply AUTHORS]: Thanks, I've updated that.

File: AUTHORS (hunk 7 above)
Comment (MaqOwais) [reply AUTHORS:1028]: You may use my full name as **Muhammed Abdul Quadir Owais** or if there is a length of character constraint _MaqOwais_ is Ok.

Comment (oscarbenjamin) [reply AUTHORS]: Thanks. Updated.

File: AUTHORS (hunk 9 above)
Comment (VBhat97) [reply AUTHORS:1041]: Hi @oscarbenjamin. My name can be recorder as Vaibhav Bhat.

Thank you!

**Review by VBhat97 — APPROVED** [review]

File: AUTHORS (hunk 9 above)
Comment (oscarbenjamin) [reply AUTHORS]: Thanks, done.

**Pull request merged**